Fixed issue identify tuples without brackets

harshsingh-24 opened the pull request:
Fix the issue #1577

faze-geek commented on tests/reference/run_dbg-test_assert_01-2f34744.stderr:
```diff
@@ -1,5 +1,2 @@
-    test()
-  File "tests/runtime_errors/test_assert_01.py", line 2
-    assert False
+warning: The `runtime stacktrace` is not enabled. To get the stacktraces, re-build LPython with `-DWITH_RUNTIME_STACKTRACE=yes`
```
Hi, are you using `MSVC `on a windows device to build ?

faze-geek replied:
I doubt that this change should not happen here. You should run `python run_tests.py -u --skip-run-with-dbg`

czgdp1807 commented:
@Thirumalai-Shaktivel Requesting you to review this as this is a parser related change.

harshsingh-24 commented:
Hey @faze-geek and @Thirumalai-Shaktivel. I have fixed all the build issues associated with Linux OS. When I executed `run_tests.py - u` it modified those files and removed the Assertions from them. Finally, my first PR of this org is ready for review ✨😁

Thirumalai-Shaktivel commented on tests/parser/tuple1.py:
```diff
@@ -35,3 +35,5 @@ def test():
     y,
 )
+
+t : tuple[i32, i32] = 2,3    # Without bracket annotated assignment of tuples
```
Can you please add a new line at the end of the file?

harshsingh-24 replied:
Added but last newline is not visible from github.

Thirumalai-Shaktivel commented on tests/reference/run_dbg-test_assert_01-2f34744.stderr:
```diff
@@ -2,4 +2,4 @@
   File "tests/runtime_errors/test_assert_01.py", line 2
     assert False
-AssertionError
+AssertionError
```
Remove all these `*.stderr` file changes.

harshsingh-24 replied:
Done

Thirumalai-Shaktivel commented:
Awesome, thanks for this fix!
I have left some comments above.

Thirumalai-Shaktivel commented:
> I would like to modify the documentation regarding this. Can you point me to the correct directory?

I think you can write this intruction in README.md: https://github.com/lcompilers/lpython#windows-1

Thirumalai-Shaktivel commented:
Cool, let's merge this PR. Thank you!

certik commented:
@harshsingh-24, thanks for the fix!

certik commented on README.md:
````diff
@@ -103,6 +103,17 @@ inst\bin\lpython examples\expr2.py -o a.out
+
+```
+python run_tests.py -u --skip-run-with-dbg
+```
````
The `--skip-run-with-dbg` flag should only be used on Windows, on Linux/macOS it should not be used.

faze-geek replied:
Yes, more specifically for the `MSVC` build setup. On windows with WSL, `./run_tests.py -u ` runs just fine.

certik replied:
I see it's in the Windows section of the README. I guess that's fine. We should eventually refactor it into a dedicated section and show commands for all platforms.
